### PR TITLE
adds api token page functional tests

### LINKED ISSUE #128: add functional tests for api token feature

### PR DESCRIPTION
**Pull request by warrenv**
Resolves #128

### Description of Changes

Adds functional tests for api token creation and deletion.
To test, run `npm run test:functional`

### PR Review Checklist

* [ ] **Have you tested the changes?**
* [ ] **Is QE review required? If so, has this been reviewed by QE?** (For all new features, review by QE is required. Minor changes which already have passing functional tests may not need a new QE review -- when in doubt, loop in a QE reviewer).
* [ ] **Are there code changes that require new unit tests? If so, are those tests included?** (A separate PR for tests is not appropriate).
* [ ] **Are documentation updates required? If so, does this PR include it?** (Separate documentation PR is generally not appropriate).
* [ ] **Are the added/edited files named appropriately and located correctly in the project directory structure?**

### CODE REVIEW

**Comment by warrenv** [comment]
Can you try running it a second time?

File: tests/functional/page-objects/token-page.js [hunk 1]
```diff
@@ -2,7 +2,20 @@ import { Selector } from 'testcafe'
 
 const PAGE_CLASS = '.CreateToken'
 const pageClass = Selector(PAGE_CLASS)
+const apiTokens = Selector(`${PAGE_CLASS} .BoxToken__table tbody tr`)
```
Comment (kennylavender) [inline tests/functional/page-objects/token-page.js]: Should we maybe add a more semantic class for individual tokens?

**Comment by warrenv** [comment]
I'm open to suggestions.

File: tests/functional/page-objects/token-page.js (hunk 1 above)
Comment (kennylavender) [reply tests/functional/page-objects/token-page.js]: `.BoxToken__table .token`?

Comment (kennylavender) [reply tests/functional/page-objects/token-page.js]: Just so its not tied to the strucutre of the table.

Comment (warrenv) [reply tests/functional/page-objects/token-page.js]: I agree with that reasoning. @wzalazar asked for a BEM formatted classname in the last PR so I'm guessing he'd want to see that here as well. I'm not too familiar with BEM so I'm not sure exactly what the name would look like.

Comment (warrenv) [reply tests/functional/page-objects/token-page.js]: ok, that works for me.

**Pull request merged**